vars/supportedUpgradeFromVersions.groovy

```diff
@@ -4,12 +4,13 @@ def call(List base_versions_list, String linux_distro, String new_scylla_repo) {
     if (base_versions_list.size() == 0) {  // auto mode, get the supported base versions list by a hydra command
         def result = sh (returnStdout: true,
                          script: """ ./docker/env/hydra.sh get-scylla-base-versions --only-print-versions true \
-                                     --linux-distro ${linux_distro} --scylla-repo ${new_scylla_repo} """)
-        println(result)
+                                     --linux-distro ${linux_distro} --scylla-repo https://s3.amazonaws.com/downloads.scylladb.com/unstable/scylla-enterprise/enterprise-2022.1/rpm/centos/2022-03/scylla.repo """)
```

Comment:
why is this URL hardcoded here?

Reply:
Good catch - leftover from some testing on Jenkins. Reverted to `${new_scylla_repo}`